angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-accordion-item/sprk-masthead-accordion-item.component.ts

```diff
@@ -1,36 +1,35 @@
 import { Component, Input, OnInit } from '@angular/core';
-import * as _ from 'lodash';
-import { toggleAnimations } from '../../sprk-toggle/sprk-toggle-animations';
+import { uniqueId } from 'lodash';
+import { toggleAnimations } from '../../../sprk-toggle/sprk-toggle-animations';
 
 @Component({
   selector: 'sprk-masthead-accordion-item',
   template: `
     <li [ngClass]="getClasses()">
-      <div
-        class="sprk-u-Position--relative sprk-o-Stack__item sprk-u-Width-100"
-      >
+      <div sprkStackItem class="sprk-c-MastheadAccordion__summary-content">
```

Comment:
Updated to use stack item directive and I made a new CSS class that applies the position relative and 100% so that we aren't the utility class.